README.md

```diff
@@ -28,7 +28,7 @@ To upgrade, simply run the `pip install` command with the `-U` option:
 ### Installing from Source
 The Viam Python SDK uses native libraries to support communication over WebRTC, which will allow you to connect to robots that are not on the same network. In order to facilitate that communication, there is a [Rust utils repo](https://github.com/viamrobotics/rust-utils) that contains the necessary protocols. Therefore, to build from source, you will need both the Rust utils and the Rust compiler.
 
-1. Download/clone this repository
+1. Download/clone [this repository] (https://github.com/viamrobotics/viam-python-sdk)
```

Comment:
```suggestion
1. Download/clone [this repository](https://github.com/viamrobotics/viam-python-sdk)
```

Reply:
thank you! should be fixed now

![Screenshot 2023-04-21 at 2 14 59 PM](https://user-images.githubusercontent.com/22106496/233706773-65b8319a-23e9-4712-bac7-db3ea103bbc4.png)